### DISCUSSION TITLE
How to extract merged text in a table?

### DISCUSSION THREAD

**Discussion by situchen** [jsvine/pdfplumber#410] [Ask for help with specific PDFs]
How to extract the merged text in the table is currently separated, for example: Base and Material, I hope these two are connected together, the expected result: Base Material. 
![image](https://user-images.githubusercontent.com/63528537/114336240-14623900-9b81-11eb-9c70-9eac2880c854.png)

**Comment by samkit-jain** [comment]
Hi @situchen Copying my answer from #285 to here as well. Using the default table extraction settings, this is the result I get
![image](https://user-images.githubusercontent.com/15127115/114392357-4772ef80-9bb6-11eb-8e7c-425c0362e56e.png)
You'll notice that there is a hidden horizontal line separator in the "Base Material" cell and that's why it is coming in 2 cells.

To discard those invisible lines, you may use the code at https://github.com/jsvine/pdfplumber/issues/311#issuecomment-726958591 It will give you the result as 
![image](https://user-images.githubusercontent.com/15127115/114393215-66be4c80-9bb7-11eb-9830-20e2ce4ca774.png)
and that should solve your problem.

**Reply by situchen** [reply]
OK，Thank you for your answers, thank you very much! @samkit-jain
